Experimental/README.md

```diff
@@ -0,0 +1,22 @@
+# ‼️ Experimental Samples! Use at your own risk ‼️
+
+The goal of this folder is to aggregate ideas and small projects to be shared with the community, without guaranteeing those are Unity recommended practices. They are there for fostering discussion and with little quality guarantee, not for production use.
+
+This code might get promoted to full bitesize samples or they also might get removed if we figure they are not promoting good practices anymore. Please be critical when looking at these samples and use these at your own risk. If you need education for game features critical to your production, don't use these samples and stick to our regular samples and docs.
+
+## Process to contribute
+
+1. Create a new pull request against main.
+1. The samples team will look at it real quick to see if there are no major blockers and publish it. No release announcements.
+1. The samples team will merge it back to develop afterward.
```

Comment:
## Contributing

1. Create a new pull request against main.
2. The samples team will review for major blockers and publish. Please note, we will not make release announcements for experimental samples.
3. The samples team will merge it back to develop afterward.